### PR TITLE
Feature/api work package embedded activities

### PR DESCRIPTION
**Pull request by mtakac2**

### CODE REVIEW

**Comment by myabc** [comment]
@mtakac I'm getting 2 spec failures when running locally:

```
Failures:

  1) API v3 Work package resource #get when acting as a user with permission to view work package should respond with 200
     Failure/Error: get get_path
     NoMethodError:
       undefined method `[]' for nil:NilClass
     # ./spec/api/work_package_resource_spec.rb:62:in `block (4 levels) in <top (required)>'

  2) API v3 Work package resource #get when acting as a user with permission to view work package should respond with work package in HAL+JSON format
     Failure/Error: get get_path
     NoMethodError:
       undefined method `[]' for nil:NilClass
     # ./spec/api/work_package_resource_spec.rb:62:in `block (4 levels) in <top (required)>'

```

same as on this previous Travis build: https://travis-ci.org/opf/openproject/jobs/28868530#L2782

**Comment by mtakac2** [comment]
@myabc That's strange - I am not getting any and CI passed..

**Comment by manwithtwowatches** [comment]
@mtakac it's in the work package representer:
{ href: "#{root_url}/api/v3/work_packages/#{represented.work_package.id}", title: "#{represented.subject}" }

ArgumentError: Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true

**Comment by manwithtwowatches** [comment]
...doesn't like root_url for some reason

**Comment by mtakac2** [comment]
Okay - Got it. Thanks for help ;)

**Comment by manwithtwowatches** [comment]
looks good to me.

File: config/environments/test.rb [hunk 1]
```diff
@@ -69,4 +69,5 @@
 
   # we use per process memory for caching in the test environment
   config.cache_store = :memory_store
+  config.action_controller.default_url_options = { host: 'localhost', port: 3000 }
```
Comment (manwithtwowatches) [inline config/environments/test.rb:72]: @mtakac I think this might be breaking test_link_to_project in application_helper_test.rb.

**Comment by mtakac2** [comment]
@myabc Okay, thanks.

**Pull request merged**